templates/shared/forms/_recaptcha.html

```diff
@@ -0,0 +1,18 @@
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<script>
+  // move g-recaptcha-response to marketo field
+  var marketoForm = document.querySelector('form#mktoForm');
```

Comment:
Could we do `document.querySelector("form[id^=mktoForm]")` instead? So we can keep form ids in the `mktoForm_<marketo ID>` format?

Reply:
If we have multiple forms on the same page, we'll also need `querySelectorAll` and a loop.

Reply:
Or use a class?

Reply:
For example: `document.querySelectorAll(".mktoForm")`

Reply:
Sure, and if we're going to do that, why not change `mkto` to `marketo`? We're not writing Go here.

Reply:
@nottrobin please don't change it in the id without a good reason though. IMHO, the amount of fixes required (eg. transitioning simple string contains to RegEx to blend new and old data) in GA, Datastudio and many spreadsheets, for the sake of not writing Go, is not worth it.

Reply:
@caldav Yeah, I can see that. I was referring to @anthonydillon's suggestion that we create a class - in which case we're choosing what to name it. If we were choosing the IDs from scratch then I would say `marketo-form` would be a better scheme, but it definitely seems like a lot of hassle to change it now. So I'm not advocating that.

Reply:
I believe @nottrobin is referring to the class name not the id.